Configuration/ProcessModifiers/python/brokenLine_cff.py

```diff
@@ -0,0 +1,5 @@
+import FWCore.ParameterSet.Config as cms
+
+# This modifier is for replacing the default pixel track "fitting" with broken line fit
+
+brokenLine =  cms.Modifier()
```

Comment:
here I think is the cpu workflow, it needs to be ported to task and then one will need a mechanism to switch workflow (if we want to keep to cpu workflows)

Reply:
btw @makortel, could you have a look or just give me few suggestions on what to do?
(I suppose copy/paste from Rienmann but at this point we may consolidate and have a simpler configurable as in the gpu case)

Reply:
it really depends if we want to continue testing and validating the two fits in parallel (and therefore support two relval workflows) or we just decide for one as default and leave to "expert" testing the comparison of the two...

for what concern the details:
```* pixelTracks.runOnGPU = True``` is in my opinion obsolete: we decided not support mixed workflows.
I personally would "consolidate"  PixelFitterByBrokenLineProducer.cc and  PixelFitterByRienmannFitProducer.cc into a single producer (as for the GPU workflow)
and have on CPU  either "legacy-fit" or "novel-fit" that can be either RF or BL.
This will also simplify the cfis and cffs